lib/ledger_sync/adaptors/quickbooks_online/adaptor.rb

```diff
@@ -96,6 +97,17 @@ def refresh!
           raise parse_error(error: e)
         end
 
+        def revoke_token!
```

Comment:
I expect methods that end in a bang to raise an error if they fail. So I'd say either do that or change to `revoke_token`

Reply:
Ah nvm, that sounds good then